src/types/plugin-input.ts

```diff
@@ -0,0 +1,21 @@
+import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as WebhookEventName } from "@octokit/webhooks";
+import { SupportedEvents } from "./context";
+import { StaticDecode, Type as T } from "@sinclair/typebox";
+
+export interface PluginInputs<T extends WebhookEventName = SupportedEvents> {
+  stateId: string;
+  eventName: T;
+  eventPayload: WebhookEvent<T>["payload"];
+  settings: CommandQuerySettings;
+  authToken: string;
+  ref: string;
+}
+
+export const commandQueryUserScheme = T.Object({
+  /**
+   * Allows any user to query anyone else. If false, only org members can query others.
+   */
+  allowPublicQuery: T.Boolean({ default: true }),
```

Comment:
Not in spec but if you default to allow everybody to query then it's fine.

Reply:
Yes default il allowed for all. Thought it would be nice to have since it queries wallets, so only admins could query them.